### ISSUE TITLE
ntsnet example not working

### ISSUE THREAD

**Issue by talhadar** [pytorch/hub#123]
Hi,
I'm trying to run the ntsnet example :
`
from torchvision import transforms
import torch
import urllib
from PIL import Image

transform_test = transforms.Compose([
    transforms.Resize((600, 600), Image.BILINEAR),
    transforms.CenterCrop((448, 448)),
    # transforms.RandomHorizontalFlip(),  # only if train
    transforms.ToTensor(),
    transforms.Normalize((0.485, 0.456, 0.406), (0.229, 0.224, 0.225)),
])

model = torch.hub.load('nicolalandro/ntsnet-cub200', 'ntsnet', pretrained=True, **{'topN': 6, 'device':'cpu', 'num_classes': 200})
model.eval()

url = 'https://raw.githubusercontent.com/nicolalandro/ntsnet-cub200/master/images/nts-net.png'
img = Image.open(urllib.request.urlopen(url))
scaled_img = transform_test(img)
torch_images = scaled_img.unsqueeze(0)

with torch.no_grad():
    top_n_coordinates, concat_out, raw_logits, concat_logits, part_logits, top_n_index, top_n_prob = model(torch_images)

    _, predict = torch.max(concat_logits, 1)
    pred_id = predict.item()
    print('bird class:', model.bird_classes[pred_id])
`

but i'm getting the following error:
`
Exception has occurred: RuntimeError
index 1623497638210 is out of bounds for dimension 1 with size 1614
  File "C:\Users\tal\.cache\torch\hub\nicolalandro_ntsnet-cub200_master\nts_net\model.py", line 61, in forward
    top_n_prob = torch.gather(rpn_score, dim=1, index=top_n_index)
  File "D:\code311016\DeepGrading\Scripts\test_ntsnet.py", line 25, in <module>
    top_n_coordinates, concat_out, raw_logits, concat_logits, part_logits, top_n_index, top_n_prob = model(torch_images)
`

**Comment by ailzhang** [comment]
Hmmm I can confirm your script(which is in the webpage) works for me both locally and it passed CI. 
Are you on latest torchvision and torch release btw?
